types/enzyme/enzyme-tests.tsx

```diff
@@ -497,9 +521,9 @@ function ShallowWrapperTest() {
 // ReactWrapper
 function ReactWrapperTest() {
     let reactWrapper: ReactWrapper<MyComponentProps, MyComponentState> =
-        mount<MyComponentProps, MyComponentState>(<MyComponent stringProp="value" numberProp={1} />);
+        mount<MyComponentProps, MyComponentState>(<MyComponent stringProp="value" numberProp={1} functionProp={() => true} />);
 
-    let reactElement: ReactElement;
+            let reactElement: ReactElement;
```

Comment:
Please revert indentation bumps like this as well

Reply:
Is there a formatter for DefinitelyTyped? Does the dtslinter have a --fix option or is there a prettier config? I was looking but didnt' see anything.

Reply:
Not that I know of :(